pkgs/tools/filesystems/mymcplus/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, fetchFromSourcehut
+, python3Packages
+, wrapGAppsHook
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mymcplus";
+  version = "3.0.5";
+
+  src = fetchFromSourcehut {
+    owner = "~thestr4ng3r";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-GFReOgM8zi5oyePpJm5HxtizUVqqUUINTRwyG/LGWB8=";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    pyopengl
+    wxPython_4_2
+  ];
+
+  meta = with lib; {
+    homepage = "https://git.sr.ht/~thestr4ng3r/mymcplus";
+    description = "A PlayStation 2 memory card manager";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ ];
```

Comment:
For new packages, we'd want maintainers to be set.

Reply:
Do you want to adopt this? I'm OK with that.

Reply:
Not for this package. I was ok adopting ukmm because I wanted to use it as a replacement for bcml, but I wouldn't actually use this.

Reply:
OK. I just changed things a bit... to get rid of Python3Packages... and to move this into python-packages so it can be rebuilt with Python10, Python11. I'm learning a bit.

Reply:
Oh you'd normally only want to add it to `pythonPackages` if it acts as a library, but in this case mymcplus is an application, so it would be best to continue with what you had before.

There's still ways to change the python version with `override`, but there's no need to have hydra actually build multiple versions of it.

Also, almost forgot, since it's an application you should be using `buildPythonApplication` instead of  `buildPythonPackage`.

If your curious to know more, here are the docs for python packaging: https://nixos.org/manual/nixpkgs/stable/#user-guide